scripts/run_mypy_checks.py

```diff
@@ -826,16 +859,31 @@ def main(args=None):
         'Installed Mypy and stubs for third party libraries.')
 
     python_utils.PRINT('Starting Mypy type checks.')
-    cmd = get_mypy_cmd(getattr(parsed_args, 'files'))
-    process = subprocess.call(cmd, stdin=subprocess.PIPE)
-
-    if process == 0:
+    cmd = get_mypy_cmd(getattr(parsed_args, 'files'), parsed_args.ci_check)
```

Comment:
Why use `getattr` instead of `parsed_args.files`?

Reply:
files won't be an attribute if `--files` is not passed. So it will throw error.

Reply:
I think `--files` has to be passed since when it's added as an argument, `nargs='+'` ([docs](https://python.readthedocs.io/en/v2.7.2/library/argparse.html#nargs)). If `--files` is omitted, argparse will throw an error, right?

More generally, I think specifying a default value is a cleaner way to handle this

Reply:
> If --files is omitted, argparse will throw an error, right?

No, the error is not thrown in this case.

Reply:
Changed it to parsed_args.files. It doesn't throw error now. Thanks!